### PR TITLE
Describe maintainer collaboration

### PR DESCRIPTION
**Pull request by milibopp**
I left out a few items that will subsequently be opened as GitHub issues.

### CODE REVIEW

**Comment by milibopp** [comment]
There is a general discussion here about where do we put this? As a placeholder, I felt the `maintainers/` folder suggested in #346 seemed appropriate at first. But some of these points overlap with what is in `CONTRIBUTING.md` already. It could become part of the contribution guides too.

**Comment by fricklerhandwerk** [comment]
@milibopp `CONTRIBUTING.md` is for contributing to documentation in the Nix ecosystem in general. That should find its own spot rendered on nix.dev itself. This stuff here is just for the documentation team itself and can stay a pure markdown file.

**Comment by milibopp** [comment]
Yup, I'm still going to integrate this into `maintainers/README.md`

File: maintainers/README.md [hunk 1]
```diff
@@ -72,18 +72,21 @@ As the Nix community is distributed globally, available time for synchronous com
 
 To keep discussions highly focused and make their results accessible, we will:
 
-- set meeting agendas on [the team's GitHub project board](https://github.com/orgs/NixOS/projects/15) from issues and pull requests
+- prepare meeting agendas as a pull requests on the [nix.dev repository](https://github.com/NixOS/nix.dev)
```
Comment (fricklerhandwerk) [inline maintainers/README.md:75]: @infinisil writes agendas as a Discourse thread, and the reply are the meeting notes

File: maintainers/README.md [hunk 2]
```diff
@@ -72,18 +72,21 @@ As the Nix community is distributed globally, available time for synchronous com
...
+- add notes to existing issues or pull requests as a comment
+- post the meeting protocol on the [Discourse documentation category](https://discourse.nixos.org/c/dev/documentation/25)
+- update calendar invitations
 
 Meeting notes should contain:
 
 - date of meeting
 - list of attendees
-- results and links to GitHub comments for agenda item.
+- results and links to GitHub issues and PRs for each agenda item
```
Comment (fricklerhandwerk) [inline maintainers/README.md:89]:
```suggestion
- results and links to GitHub issues and pull requests for each agenda item
```

**Review by fricklerhandwerk — APPROVED** [review]

**Pull request merged**